modules/exploits/windows/misc/manageengine_eventlog_analyzer_rce.rb

```diff
@@ -0,0 +1,219 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
```

Comment:
Even when it's trying to be careful it is doing some mods in the target database which I'm concerned about, kind of `SELECT lo_unlink(-1)`. I'd recommend to use ManualRanking here.